src/Language/PureScript/Sugar/Names/Env.hs

```diff
@@ -60,16 +60,16 @@ data Imports = Imports
   , importedModules :: S.Set ModuleName
   -- |
   -- The names of "virtual" modules that come into existence when "import as"
-  -- is used.
+  -- is used, mapped to the real module being imported.
   --
-  , importedVirtualModules :: S.Set ModuleName
+  , importedVirtualModules :: M.Map ModuleName ModuleName
```

Comment:
Oh, yes, of course. Although maybe a `[ModuleName]` would be sufficient? I think we can look in the other fields of the `Imports` value for that information?